js/backbone.upload-manager.js

```diff
@@ -205,6 +205,7 @@
                 while (self.files.length) {
                     self.files.at(0).cancel();
                 }
+                self.$('#' + self.options.fileUploadId).val('');
```

Comment:
Just removed the _clearPreviousAddFileInput function and directly set to empty value.